tests/rptest/tests/admin_api_raft_metrics_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.admin import Admin
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.util import wait_until_result
+from rptest.util import wait_until
+import requests
+
+# Errors we should tolerate right after start Redpanda
+NOT_LEADER_LOG_ERRORS = [
+    # e.g.  admin_api_server - admin_server.cc:409 - [_anonymous] exception intercepted - url: [http://docker-rp-6:9644/v1/debug/raft/kafka/topic-sppnwqzxzz/0] http_return_status[500] reason - seastar::httpd::server_error_exception (admin_server.cc:3347 - Can not find leader for ntp {kafka/topic-sppnwqzxzz/0})">
+    "admin_api_server - .*http_return_status[500]"
```

Comment:
i think this exception should be mapped to 503 and not logged with an error severity